documentation/modules/ROOT/pages/transformations/mongodb-event-flattening.adoc

```diff
@@ -253,58 +444,115 @@ Because the structure of these other messages differs from the structure of the
 
 For more information about how to apply the SMT selectively, see xref:{link-smt-predicates}#applying-transformations-selectively[Configure an SMT predicate for the transformation].
 
+// Type: reference
+// ModuleID: configuration-options-for-the-debezium-mongodb-event-flattening-transformation
+// Title: Configuration options for the {prodname} event flattening transformation for MongoDB
 [[mongodb-extract-new-record-state-configuration-options]]
 == Configuration options
+
+The following table describes the configuration options for the MongoDB event flattening SMT.
+
 [cols="30%a,25%a,45%a"]
 |===
 |Property |Default |Description
 
 |[[mongodb-extract-new-record-state-array-encoding]]<<mongodb-extract-new-record-state-array-encoding, `array.encoding`>>
 |`array`
-|The SMT converts MongoDB arrays into arrays as defined by Apache Connect (or Apache Avro) schema.
+|Specifies the format that the SMT uses when it encodes arrays that it reads from the original event message.
+Set one of the following options:
+
+`array`::
+The SMT uses the array datatype to encode arrays.
+To ensure that downstream clients can process the message, all elements in each array instance must be of the same type.
+
+ format that is compatible with Apache Kafka Connect or Apache Avro schemas.
```

Comment:
This looks like a text left from previous version.
Also the other option `struct` should be mentioned.

Reply:
Oh! Sorry about that. Looks like I walked away mid-sentence and never returned. 😨 

Fixed now.